engine/installation/linux/ubuntulinux.md

```diff
@@ -49,64 +49,90 @@ your `APT` sources to the new Docker repository.
 ### Update your apt sources
 
 Docker's `APT` repository contains Docker 1.7.1 and higher. To set `APT` to use
-packages from the new repository:
+packages from the Docker repository:
 
-1. Log into your machine as a user with `sudo` or `root` privileges.
+1.  Log into your machine as a user with `sudo` or `root` privileges.
 
-2. Open a terminal window.
+2.  Open a terminal window.
 
-3. Update package information, ensure that APT works with the `https` method, and that CA certificates are installed.
+3.  Update package information, ensure that APT works with the `https` method, and that CA certificates are installed.
 
-         $ sudo apt-get update
-         $ sudo apt-get install apt-transport-https ca-certificates
+    ```bash
+    $ sudo apt-get update
+    $ sudo apt-get install apt-transport-https ca-certificates
+    ```
+4.  Add the new `GPG` key.
 
-4. Add the new `GPG` key.
+    ```bash
+    $ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+    ```
 
-        $ sudo apt-key adv --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
-
-5. Open the `/etc/apt/sources.list.d/docker.list` file in your favorite editor.
+5.  Open the `/etc/apt/sources.list.d/docker.list` file in a text editor.
```

Comment:
for a follow up, we could consider just showing `echo deb ...... > /etc/apt/sources.list.d/docker.list`